subsys/net/ip/net_tc.c

```diff
@@ -18,6 +18,8 @@ LOG_MODULE_REGISTER(net_tc, CONFIG_NET_TC_LOG_LEVEL);
 #include "net_stats.h"
 #include "net_tc_mapping.h"
 
+#define MAX_NAME_LEN sizeof("xx_q[y]")
```

Comment:
Feels like a comment here about what 'xx' means would be good.

Reply:
Just a normal design pattern to calculate the output string length. The "xx" in this context means either "rx" or "tx" in the output string.